README.md

```diff
@@ -394,6 +394,55 @@ Set the fields for the CSV export function. By default, ng-admin uses the fields
                 .stripTags(true)
         ]);
 
+
+### batchDeleteView Settings
+
+* `singleApiCall(function(entityIds) {})`
+
+By default, the batch delete action make as many requests as you selected objects. If you want to delete all selected objects in a single request, you have to define a `singleApiCall` function. You can use it if you API support filter for multiple values on a DELETE request.
+
+        // Will call DELETE /posts?_filters=id1,id2,.._
+        post.batchDeleteView()
+            .singleApiCall(function (ids) {
+            return postIds.join(,);
```

Comment:
Why don't you use the same syntax as the `singleApiCall()` for references, where the function returns an object litteral? this would prevent the use of a request interceptor.